distribution/changelog.txt

```diff
@@ -33,6 +33,7 @@
 - Fix: [#22339] Printing ui.tool.cursor in console crashes the game.
 - Fix: [#22348] Progress bar screen doesn’t handle window resizing.
 - Fix: [#22389] Alpine coaster has wrong tunnel entrance type.
+- Fix: [#22435] [Plugin] Fix off-by-one pixel issue in active widget width and height setters.
```

Comment:
List the bug, not the fix.

Reply:
Fixed fix to non-fix in fix in fixes changelog. 😃